packages/flutter/lib/src/widgets/text_editing_action.dart

```diff
@@ -0,0 +1,70 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+import 'actions.dart';
+import 'editable_text.dart';
+import 'focus_manager.dart';
+import 'framework.dart';
+
+// Similar to CallbackAction's OnInvokeCallback, but includes EditableTextState.
+typedef _OnInvokeTextEditingCallback<T extends Intent> = Object? Function(T intent, EditableTextState editableTextState);
+
+/// An [Action] related to editing text.
+///
+/// If an [EditableText] is currently focused, then _editableTextState will be
+/// set and the given [onInvoke] callback will be called with it. If not, then
+/// [isEnabled] will be false and [onInvoke] will not be called.
+///
+/// The focused [EditableText] must have a [Key]. This is handled automatically
```

Comment:
Is any Key sufficient?

Reply:
Actually, from the code below it looks like you'll need a global key?

Reply:
Yes, I'll update the docs.